src/Microsoft.AspNet.Http/ConnectionInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.Http
+{
+    public abstract class ConnectionInfo
```

Comment:
One reason that I remember we choose this was so that we could more easily add new (virtual) items without them being official breaking changes.